src/Redux/Books/bookTypes.js

```diff
@@ -0,0 +1,2 @@
+export const ADD_BOOK = 'ADD_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Hi @aamirkhan2478, I think defining action type strings in a separate file is not a good coding practice. You should place these action types within your action creators and one or more of your reducers
> **Explanation**
  > - Defining the action types as constants and importing them wherever used reduces the risk of accidentally misspelling them in one of the sources in which they're used, which can cause a lot of wasted time debugging a simple bug/misspelling. Hence kindly move these action types into the `books.js` file where you are using them.

- [ ] Secondly based on the requirements of this project, your action types should follow the [duck pattern](https://github.com/erikras/ducks-modular-redux) methodology. This is very important especially if you have plans to scale your project in the future. Hence you'll need something like the line of code below:

```.ruby
export const ADD_BOOK = 'bookstore/books/ADD_BOOK';

```